js/__tests__/index.test.js

```diff
@@ -18,4 +18,29 @@ describe('SentryCli', function() {
       expect(e.message).toMatch('Command failed:');
     });
   });
+
+  // test('call sentry-cli with source map options', function() {
```

Comment:
Was this meant to be commented in the final PR?

Reply:
> Don't worry about the tests, we will do that afterwards. Just leave the comment in. However, you can test locally by calling npm install, which will also download the current binary.

Reply:
ah, sorry was just curious.

Reply:
The first test doesn't make sense either... :D